logos-codegen/src/lib.rs

```diff
@@ -240,7 +240,8 @@ pub fn generate(input: TokenStream) -> TokenStream {
                     let allow_greedy = definition.allow_greedy.unwrap_or(false);
                     if !allow_greedy && pattern.check_for_greedy_all() {
                         parser.err(concat!(
-                            "This pattern contains an unbounded greedy dot repetition (.* or .+). ",
+                            "This pattern contains an unbounded greedy dot repetition, i.e. `.*` or `.+` ",
+                            "(or a character class that is equivalent to a dot, i.e. `[^\\n]*`). ",
```

Comment:
```suggestion
                            "(or a character class that is equivalent to a dot, i.e., `[^\\n]*`). ",
```

Commas after `i.e.` are debatable, but are usually favored :-)